starter-code/InvestingPortfolio/app/src/main/java/drewmahrt/generalassemb/ly/investingportfolio/MainActivity.java

```diff
@@ -24,43 +27,187 @@
 import com.android.volley.toolbox.JsonArrayRequest;
 import com.android.volley.toolbox.JsonObjectRequest;
 import com.android.volley.toolbox.Volley;
+import com.google.android.gms.appindexing.Action;
+import com.google.android.gms.appindexing.AppIndex;
+import com.google.android.gms.appindexing.Thing;
+import com.google.android.gms.common.api.GoogleApiClient;
 
 import org.json.JSONArray;
 import org.json.JSONException;
 import org.json.JSONObject;
 
-public class MainActivity extends AppCompatActivity implements LoaderManager.LoaderCallbacks<Cursor>{
+import java.io.IOException;
+import java.util.ArrayList;
+import java.util.List;
+
+import okhttp3.Call;
+import okhttp3.Callback;
+import okhttp3.OkHttpClient;
+
+public class MainActivity extends AppCompatActivity implements LoaderManager.LoaderCallbacks<Cursor> {
+
     private static final String TAG = "MainActivity";
+    public static final String BASE_URL = "http://dev.markitondemand.com/Api/v2/lookup/json?input=";
+
     public static final int LOADER_STOCK = 0;
+    public static final Uri CONTENT_URI = StockPortfolioContract.Stocks.CONTENT_URI;
+
 
     RecyclerView mPortfolioRecyclerView;
     StockRecyclerViewAdapter mAdapter;
 
+    private ContentResolver mContentResolver;
+
+    private List<Stock> mStocks;
+    /**
+     * ATTENTION: This was auto-generated to implement the App Indexing API.
+     * See https://g.co/AppIndexing/AndroidStudio for more information.
+     */
+    private GoogleApiClient client;
```

Comment:
This shouldn't be in here. If you accidentally hit whatever keyboard shortcut inserts this (it's easy to hit, unfortunately) then just hit ctrl-z right away to get rid of it.